docs/source/advanced_usage.rst

```diff
@@ -196,102 +183,116 @@ list of qubits to build your program.
 Classical Control Flow
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Here are a couple quick examples that show how much richer the classical
```

Comment:
add a note about its use on a real QPU? (i.e., that it's not supported)

Reply:
good call